.github/workflows/update-document.yaml

```diff
@@ -0,0 +1,75 @@
+name: Update document
+
+on:
+  push:
+    branches:
+      - develop
+    paths:
+    - 'backend/oas/provider/**'
+    - 'backend/oas/user/**'
+    - 'terraform/infrastructure/modules/**'
+    - 'terraform/service/modules/**'
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  update-document:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+
+    - name: Set up Git
+      run: |
+        git config --global user.name "github-actions[bot]"
+        git config --global user.email "github-actions[bot]@users.noreply.github.com"
+  
+    - name: Set up Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: 3.12.4
+
+    - name: Install Poetry
+      run: |
+        pip install poetry poetry-plugin-export
+        echo "$HOME/.local/bin" >> $GITHUB_PATH
+
+    - name: Install poetry dependencies
+      run: poetry install
+
+    - name: Install make
+      run: sudo apt install make
+
+    - name: Set up Aqua
+      run: |
+        make setup-aqua
+        make doctor
+
+    - name: Update document
+      run: make docs
+
+    - name: Create new branch and switch
+      run: |
+        BRANCH_NAME="chore/auto-update-docs-$(date +%Y%m%d-%H%M%S)"
+        git checkout -b $BRANCH_NAME
+        echo "BRANCH_NAME=$BRANCH_NAME" >> $GITHUB_ENV
+
+    - name: Auto PR
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        TRIGGERED_PUSH_URL: ${{ github.server_url }}/${{ github.repository }}/commit/${{ github.sha }}
+      run: |
+        if git diff --quiet; then
+          echo "No changes to commit."
+          exit 0
+        fi
+        git commit -m "Auto update document $(date +%Y%m%d-%H%M%S)" -a
+        git push origin $BRANCH_NAME
+        gh pr create --title "chore: [CI] Auto update document $(date +%Y%m%d-%H%M%S)" \
+                      --body "This PR was automatically generated.$TRIGGERED_PUSH_URL" \
+                      --base develop \
+                      --head $BRANCH_NAME \
+                      --label documentation
```

Comment:
@dvlpwork 
Thank you! 🎉 
Please adding new line. 🙏

Reply:
As another task, it is good if following comment have done by [reviewdog](https://github.com/reviewdog/reviewdog)

> @dvlpwork
> Thank you! 🎉
> Please adding new line. 🙏